smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolUnitTestResponseGenerator.kt

```diff
@@ -172,7 +171,7 @@ open class HttpProtocolUnitTestResponseGenerator protected constructor(builder:
                 writer.openBlock("public static func ==(lhs: \$L, rhs: \$L) -> Bool {", "}", symbol.fullName, symbol.fullName) {
                     when (shape) {
                         is StructureShape -> {
-                            shape.members().filter { !it.hasTrait<HttpQueryTrait>() }.forEach { member ->
+                            shape.members().forEach { member ->
```

Comment:
Members with the `@httpQuery` trait are no longer excluded when comparing equality in response tests.